### PR TITLE
Update the conda environments and README file

### PR DESCRIPTION
**Pull request by BradReesWork**
updated the conda environments to have cmake 3.18 and update the README file

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

**Review by ajschmidt8 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,6 +30,17 @@ for i in range(len(df_page)):
 		" PageRank is " + str(df_page['pagerank'].iloc[i]))
 ```
 
+## Getting cuGraph
+There are 3 ways to get cuGraph :
+1. [Quick start with Docker Demo Repo](#quick)
```
Comment (rlratzel) [inline README.md:35]: I don't think they're calling it a demo repo anymore, so I would just change it to "Quick start with the RAPIDS Docker repo"

File: README.md [hunk 2]
```diff
@@ -85,13 +95,13 @@ for i in range(len(df_page)):
...
-| _Multigraph_      | _coming in 0.18_                                      |
-| _MultiDigraph_    | _coming in 0.18_                                      |
+| Multigraph      | A Graph with multiple edges between a vertex pair   |
+| MultiDigraph    | A Directed Graph with multiple edges between a vertex pair   |
 |  |  |
 
 </br></br>
 ## Supported Data Types
-cuGraph supports the creation of a graph several data types:
+cuGraph supports graph creation with Source and Distentaion being expressed as:
```
Comment (rlratzel) [inline README.md:104]: typo: "Destination"

File: README.md [hunk 3]
```diff
@@ -125,22 +135,14 @@ The amount of memory required is dependent on the graph structure and the analyt
...
-1. [Quick start with Docker Demo Repo](#quick)
-2. [Conda Installation](#conda)
-3. [Build from Source](#source)
-
-
-
-
+---
 ## Quick Start <a name="quick"></a>
 Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapidsai/), choosing a tag based on the NVIDIA CUDA version you’re running. This provides a ready to run Docker container with example notebooks and data, showcasing how you can utilize all of the RAPIDS libraries: cuDF, cuML, and cuGraph.
```
Comment (rlratzel) [inline README.md:142]: not in your PR, but maybe "RAPIDS Docker Repository" instead of "demo"?

**Review by rlratzel — CHANGES_REQUESTED** [review]
The only thing that must be fixed is the typo, others are just suggestions.

File: README.md (hunk 1 above)
Comment (BradReesWork) [reply README.md:35]: updated in latest push

File: README.md (hunk 2 above)
Comment (BradReesWork) [reply README.md:104]: thanks - updated

File: README.md (hunk 3 above)
Comment (BradReesWork) [reply README.md:142]: fixed

**Review by rlratzel — APPROVED** [review]

**Review by afender — APPROVED** [review]

**Comment by BradReesWork** [comment]
@gpucibot merge

**Pull request merged**